avo-hq/avo
# refactor: support optional policy scope

# Description


Supports the scenario where you can not have a `Scope` present in a `Policy`.

From [here](https://discord.com/channels/740892036978442260/740893011994738751/1029437719866245210).

# Checklist:




## Manual review steps


1. Remove all traces of `Scope` class including `ApplicationPolicy`
1. Go to users and observe they are displayed and no errors are raised

Manual reviewer: please leave a comment with output from the test if that's the case.
